source/ui/PGM/PGMBPMStatusView.cpp

```diff
@@ -0,0 +1,162 @@
+#include "PGMBPMStatusView.h"
+#include "beamline/PGM/PGMBeamline.h"
+#include "beamline/PGM/PGMBPMControl.h"
+
+#include <QLabel>
+#include <QVBoxLayout>
+
+PGMBPMStatusView::PGMBPMStatusView(QWidget *parent) :
+    QWidget(parent)
+{
+
+    // Set widgets and layout for PGMBpmControls
+	QLayout *bpmLayout = setupBPMDisplay();
+
+    // Main layout
+    QVBoxLayout *mainLayout = new QVBoxLayout;
+	mainLayout->addLayout(bpmLayout);
+
+    setLayout(mainLayout);
+
+	// Set connections for PGMBpmContols
+	setupBPMConnections();
+}
+
+QLayout * PGMBPMStatusView::setupBPMDisplay(){
+
+    bpm10IDvalueX_ = new QLabel("   um");
```

Comment:
@iainworkman it could be. This was the original changes @dleacock made. It would be great if we can use some standard AM UI components. But we can live like this so far.